handbook: improve CS new hire onboarding content (#16997)
PostHog/posthog.com: contents/handbook/cs-and-onboarding/new-hire-onboarding.md

Onboarding here is mostly self-serve - we won't sit you in a room for training for two weeks, and unlike a lot of companies, we'd prefer you get up and running with your book of customers quickly. If you're not sure who's supposed to make something below happen, the person responsible is almost certainly you.

What follows is a rough plan for your first month - use it as a guide, not a contract. The handbook itself is a work in progress, so you'll find gaps as you ramp, things you needed to know that weren't written down. That's normal, and when you find a gap your job is to fill it in so the next person has it easier.

## Week 1 – systems, demo, immersion

Week 1 is about getting set up and immersed in how we work with customers. Don't worry about being productive yet.

**Focus on:**

- Setting up the day-to-day tools you'll be using - Vitally, Gong, Slack, Metabase. See [sales and CS tools](/handbook/growth/sales/sales-and-cs-tools).
- Preparing your PostHog demo.
- Picking a recent customer call on BuildBetter to watch, and asking team members to add you to as many of their live calls as you can - the goal is exposure to how PostHog CSMs talk to customers.

> - Nailing the product fundamentals - work through the [curriculum](#posthog-curriculum) further down this page and [getting started with customers](/handbook/cs-and-onboarding/getting-started-with-customers).

Reviewer: I think we should clearly define "product fundamentals" so we can help them focus the scope of their initial learning period: 
- Events
- Persons
- Session Replay
- Feature Flags 
- Experiments
- Product Analytics
- and... MCP (thinking about 2030-shaped software, imo we should consider this as a fundamental way of interacting with PH and preemptively become experts on it for our customers)

Author: good call out! i've updated this line to define product fundamentals as:
- events, persons, product analytics being the fundamental
- session replay, feature flags, experiments as next priority, with flexibility to go deep on other products depending on customer book
- and also called out MCP

showTitle: true
---

Welcome to the PostHog Customer Success team! We only hire about 1 in 400 applicants, so you've done well to make it here!

Onboarding here is mostly self-serve - we won't sit you in a room for training for two weeks, and unlike a lot of companies, we'd prefer you get up and running with your book of customers quickly. If you're not sure who's supposed to make something below happen, the person responsible is almost certainly you.

Below is a rough plan for your first month - use it as a guide, not a contract. The handbook itself is a work in progress, so you'll find gaps as you ramp up, things you needed to know that weren't written down. That's normal, and when you find a gap your job is to fill it in so the next person has it easier.

## Week 1 – how we talk about PostHog

This week is about getting set up and learning how we talk about PostHog. You'll feel extremely unproductive, and that's fine - the aim is to set yourself up for in-person onboarding in Week 2. Read everything you can, work through the product fundamentals, and come to Week 2 with questions we can work through together.

**Focus on:**

> - Setting up the day-to-day tools you'll be using - Vitally, Gong, Slack, Metabase. See [sales and CS tools](/handbook/growth/sales/sales-and-cs-tools).

Reviewer 2: ```suggestion
- Setting up the day-to-day tools you'll be using - Vitally, Gong, Slack, Metabase. See [sales and CS tools](/handbook/growth/sales/sales-and-cs-tools) for set-up, and start copying other CSM's views and automations (or even better, build your own!)
```

- Reading the [CS](/handbook/cs-and-onboarding/) and [sales](/handbook/growth/sales/) sections of the handbook.
- Preparing your PostHog demo.
- Picking a recent customer call on Gong/BuildBetter to watch, and asking team members to add you to as many of their live calls as you can - the goal is exposure to how we talk about PostHog and how we talk to customers.

> - Nailing the product fundamentals - use the [framework](#learning-posthog) as a guide and work through the [onboarding exercise](/handbook/cs-and-onboarding/new-hire-onboarding-exercise).

Reviewer 2: ```suggestion
- Nailing the product fundamentals - use the [framework](#learning-posthog) as a guide and work through the [onboarding exercise](/handbook/cs-and-onboarding/new-hire-onboarding-exercise). Use the demo environments or your own PostHog project to test different features and use cases.
```

- Exploring Slack. We're public by default, so Slack is one of the richest resources you have. You'll find outreach messages that worked, prior conversations with customers, PostHog history on decisions like pricing changes, and context that didn't make it into the handbook.

**How to think about each product.** As you go through the fundamentals, for each product you're trying to be able to answer:

> - The value add - why a customer would care

Reviewer 2: ```suggestion
- The value add - why a customer would care
- Common use cases to demo

```

- How it's implemented and implementation quirks worth knowing
- Signs of poor implementation
- How it works alongside other PostHog products
- How to get value out of it
- What you can do with the MCP

## Week 2 – in-person onboarding, your customers

In-person onboarding typically happens this week (3-4 days led by your team lead). You'll get your book of customers, work through how to prioritise it, start digging in, and see how all the systems we use come together.

**Focus on:**

> - Walking through your book of business with Dana - prioritisation, where to start, who to reach out to first.

Reviewer 2: ```suggestion
- Walking through your book of business with your team lead - prioritisation, where to start, who to reach out to first.
```

**Focus on:**

- Reaching out to customers in your book to establish contact, scheduling handovers for any coming from existing owners. [Getting started with customers](/handbook/cs-and-onboarding/getting-started-with-customers) covers the playbook for early outreach and intro calls.
- Picking up customer questions and tickets as they come in - this is where the bulk of customer work happens. See [handling customer issues](/handbook/cs-and-onboarding/handling-customer-issues) for how tickets flow.
- Evaluating implementations as you go - is the customer set up well, are they getting value? The [foundation check](/handbook/cs-and-onboarding/foundation-check) and [health check](/handbook/cs-and-onboarding/health-checks) are the structured ways to do this.
- Refining your demo with each conversation.
- Starting to respond to Vitally signals on your book - what triggers your attention, what's the right next move? You'll feel confused at first, and that's fine, the goal is to start building a feel for it. *[link: responding-to-signals.md when written]*

> **Ship your first handbook PR.** Somewhere in here you'll hit something that wasn't documented, or that's documented wrong. Write it up and open a PR. The point isn't the PR itself, it's that the handbook only stays useful if everyone adds to it. Not knowing something isn't a failing, but leaving it undocumented for the next person is.

Reviewer 2: ```suggestion
**Ship your first handbook PR.** Somewhere along the way you'll find a gap or mistake in the handbook, or want to add a new page entirely. Write it up and open a PR. The point isn't the PR itself, it's that the handbook only stays useful if everyone adds to it. Not knowing something isn't a failing, but leaving it undocumented for the next person is.
```

> 
> ## What good looks like at the end of week 4
> 
> This is the bar for end of month 1.
> 
> **You have a clear read on your book:**
> 
> - Who's responsive and who isn't
> - Who you'd build a strong relationship with
> - Who knows you're there
> - Where the expansion opportunities sit
> 
> **You're up and running with customers:**
> 
> - Handling tickets and customer questions across the products your book uses, with help when needed
> - Doing demos for those products
> - Evaluating customer implementations and seeing where to improve them
> - Understanding how pricing works and the levers you have for cost optimization
> 
> **You're working with Vitally signals.** You know what the signals are and have a sense of how to prioritise and work through them.
> 

Reviewer 2: ```suggestion
## What good looks like at the end of week 4

This is the bar for end of month 1.

**You have a clear read on your book:**

- Who's responsive and who isn't
- Who you'd build a strong relationship with
- Who knows you're there
- Where the expansion opportunities sit

**You're up and running with customers:**

- Made contact with every customer in your book of business
- Handling tickets and customer questions across the products your book uses, with help when needed
- Doing demos for those products
- Evaluating customer implementations and seeing where to improve them
- Understanding how pricing works and the levers you have for cost optimization

**You're working with Vitally signals.** You know what the signals are and have a sense of how to prioritise and work through them.

**You're sharing with the team.** You're posting wins, learnings, opportunities for feedback, and anything else valuable in our shared channels. You were hired because we think you can improve our team, so don't be afraid to share opinions and approaches.

```

## Month 2 and beyond

**By the end of month 2:**

> - Saved your first 'we're going to churn' - it's going to happen, but you're going to save them!
> - Be independently working with your entire book to solve tricky technical problems with minimal assistance

Reviewer 2: ```suggestion
- Saved your first 'we're going to churn' - it's going to happen, but you're going to save them!
- Be independently working with your entire book to solve tricky technical problems with minimal assistance
- Focusing strategically on your customers - engaging with accounts based on risk and growth opportunity, not just reacting to tickets and signals

```
